include/net/ethernet.h

```diff
@@ -258,6 +258,9 @@ struct ethernet_lldp {
 
 	/** LLDP TX timeout */
 	u32_t tx_timer_timeout;
+
+	/** LLDP RX callback function */
+	net_lldp_recv_callback_func_t cb;
```

Comment:
There seems to be two ways the function callbacks are named in include/net, minority seems to have this "...func_t" style and most are using "...cb_t" style. Lets use the "...cb_t" style here too. So the type would be called "net_lldp_recv_cb_t" here.

Reply:
OK, sounds good